api/v1alpha1/groupversion_info.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package v1alpha1 contains API Schema definitions for the oadp.openshift.io v1alpha1 API group
+// Package v1alpha1 contains API Schema definitions for the nac v1alpha1 API group
```

Comment:
Should this be `for the nac in the oadp.openshift.io` API group? `v1alpha1` is only version of the API

Reply:
I do think so, `oadp.openshift.io` is the domain, `nac` is the group (and `v1alpha`is version). But this is generated string

Reply:
+1 to what @mpryc is saying, @mateusoliveira43 what do you mean by domain ? The group is `nac.oadp.openshift.io` , version is `v1apha1`.
Ref link: https://github.com/migtools/oadp-non-admin/blob/1d7eac38b853fd776cf4a7a045f760b2710ac53e/api/v1alpha1/groupversion_info.go#L29

Reply:
looking the PROJECT file in the root of the repo

the string may be wrong, but its generated using PROJECT.group and PROJECT.version

group from kubernetes is generated using PROJECT.group + PROJECT.domain